sql/maintenance_utils.sql

```diff
@@ -62,19 +63,18 @@ CREATE OR REPLACE FUNCTION _timescaledb_functions.get_compressed_chunk_index_for
 -- Parameters:
 --   chunk: Chunk to recompress.
 --   if_not_compressed: Print notice instead of error if chunk is already compressed.
-CREATE OR REPLACE PROCEDURE @extschema@.recompress_chunk(chunk REGCLASS,
-                                             if_not_compressed BOOLEAN = false)
+CREATE OR REPLACE FUNCTION _timescaledb_functions.recompress_chunk(chunk REGCLASS) RETURNS REGCLASS
```

Comment:
Since this is a public API wouldn't be good to add an wrapper emiting a WARNING message about the deprecation??